Cargo.toml

```diff
@@ -9,4 +9,8 @@ members = [
     "proptest-derive",
 ]
 
-exclude = ["proptest/test-persistence-location/*"]
+exclude = [
```

Comment:
Why are we excluding these?

Reply:
For kani, we exclude to avoid nested workspaces.

For examples, they no longer compile since the main library has been turned off.